src/legacy/core_plugins/kibana/public/discover/index.html

```diff
@@ -91,12 +91,25 @@
             </button>
 
             <div class="dscResultCount">
-              <span data-test-subj="discoverQueryHits" class="kuiLocalBreadcrumb__emphasis">{{(hits || 0) | number:0}}</span>
+              <strong data-test-subj="discoverQueryHits">{{(hits || 0) | number:0}}</strong>
               <span
+                class="dscResultHits"
                 i18n-id="kbn.discover.hitsPluralTitle"
                 i18n-default-message="{hits, plural, one {hit} other {hits}}"
                 i18n-values="{ hits }"
               ></span>
+              <button	
+                ng-if="opts.savedSearch.id"
+                class="kuiLink"	
+                type="button"	
+                id="reload_saved_search"	
+                aria-label="{{::'kbn.discover.reloadSavedSearchAriaLabel' | i18n: {defaultMessage: 'Reload saved search'} }}"	
+                tooltip="{{::'kbn.discover.reloadSavedSearchTooltip' | i18n: {defaultMessage: 'Reload saved search'} }}"	
```

Comment:
Unnecessary to add `tooltip` and `aria-label` because the button's text is fully visible.